monai/transforms/transforms.py

```diff
@@ -214,13 +214,16 @@ def __call__(self, filename):
             img = correct_nifti_header_if_necessary(img)
             header = dict(img.header)
             header['filename_or_obj'] = name
-            header['affine'] = img.affine
+            header['original_affine'] = img.affine.copy()
+            header['affine'] = img.affine.copy()
```

Comment:
Maybe we just need to copy for "original_affine"?

Reply:
that's fixed, thanks